sapi/cgi/cgi_main.c

```diff
@@ -1005,12 +1007,6 @@ static sapi_module_struct cgi_sapi_module = {
 };
 /* }}} */
 
-/* {{{ arginfo ext/standard/dl.c */
-ZEND_BEGIN_ARG_INFO(arginfo_dl, 0)
-	ZEND_ARG_INFO(0, extension_filename)
-ZEND_END_ARG_INFO()
-/* }}} */
-
 static const zend_function_entry additional_functions[] = {
```

Comment:
Any idea what's up with these two separate function lists in this SAPI?

Reply:
huh, no idea. Although, what I can see is:
- some other SAPIs also have "additional functions"
- they all do something like this: `cgi_sapi_module.additional_functions = additional_functions;`